packages/twenty-server/src/modules/workflow/workflow-executor/utils/variable-resolver.util.ts

```diff
@@ -0,0 +1,76 @@
+import { isNil, isString } from '@nestjs/common/utils/shared.utils';
+
+import Handlebars from 'handlebars';
+
+import {
+  WorkflowExecutorException,
+  WorkflowExecutorExceptionCode,
+} from 'src/modules/workflow/workflow-executor/exceptions/workflow-executor.exception';
+
+const VARIABLE_PATTERN = RegExp('\\{\\{(.*?)\\}\\}', 'g');
+
+export const resolve = (
```

Comment:
we need some unit tests to make sure it works properly AND document how it works

Reply:
Done!